CHANGELOG.rst

```diff
@@ -10,7 +10,7 @@ v1.0.1
 Release Summary
 ---------------
 
```

Comment:
Release Summary
---------------

v1.0.1
====

Removes use of ansible.netcommon for IP address in favor of Python 3 built in ipaddress module.